app/config/config_dev.yml

```diff
@@ -6,6 +6,8 @@ framework:
         resource: "%kernel.root_dir%/config/routing_dev.yml"
         strict_requirements: true
     profiler: { only_exceptions: false }
+    validation:      { enable_annotations: true }
+    translator: { fallback: en }
```

Comment:
will not be available in prod environment (better to include it simply in config.yml)